dev-util/android-sdk-update-manager/android-sdk-update-manager-26.1.1.ebuild

```diff
@@ -1,82 +1,97 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI="8"
+PYTHON_COMPAT=(python3_{9..11})
 
-inherit desktop udev
-
-ANDROID_SDK_DIR="/opt/android-sdk-update-manager"
-MY_P="android-sdk_r${PV}-linux"
+inherit python-r1 udev distutils-r1
 
 DESCRIPTION="Open Handset Alliance's Android SDK"
 HOMEPAGE="https://developer.android.com"
-SRC_URI="https://dl.google.com/android/${MY_P}.tgz"
-S="${WORKDIR}/android-sdk-linux"
+SRC_URI="https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip"
+RESTRICT="mirror"
 
 LICENSE="android"
 SLOT="0"
+IUSE="udev"
 KEYWORDS="~amd64 ~x86"
-RESTRICT="mirror"
 
-DEPEND="acct-group/android"
+DEPEND="
+	acct-group/android
+	app-arch/gzip
+	app-arch/unzip
+	app-arch/tar
+"
+
 RDEPEND="
-	${DEPEND}
-	dev-java/ant-core
 	dev-java/swt:3.7[cairo]
-	>=virtual/jdk-1.8
+	>=dev-java/ant-core-1.6.5
 	sys-libs/ncurses-compat:5[abi_x86_32(-)]
 	sys-libs/zlib[abi_x86_32(-)]
+	virtual/jdk
```

Comment:
Please add a minimum version, never use `virtual/jdk` as a dependency without a version
https://wiki.gentoo.org/wiki/Java_Developer_Guide#JDK.2FJRE

You can use `>=virtual/jdk-1.8:*` if you know it will also work with newer java versions or `virtual/jdk:1.8` to pin it to 1.8 only